public/js/webcam.js

```diff
@@ -29,12 +31,13 @@ async function init() {
 function handleSuccess(stream) {
     window.stream = stream;
     video.srcObject = stream;
+    beforeCaptureGroup.style.removeProperty('display');
```

Comment:
Don't want user pressing capture before the stream has loaded

Reply:
OH YES, GOOD CATCH